i18n/locales/en/translation.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Why not just call it `i18n/locales/en.json`? 😉

Reply:
I used the same structure as in this [example](https://github.com/i18next/react-i18next/blob/master/example/react/public/locales/en/translation.json) from `react-i18next`. 

I guess the only benefit is that we could split our translations into multiple files for different namespaces,  e.g. have a file called `translation.json` and another one called `common.json` where the naming represents the namespace so that we would have a 'cleaner' structure. 

But I don't know if we want to do this anyways so I don't mind if we chose a different path here.

Reply:
Yeah, then let's put everything in a `en.json` for now – we can still split it up in multiple files later if it grows too large (and we shouldn't yet try to guess how we are gonna split it up later).